addons/mail/static/src/components/attachment_image/attachment_image.js

```diff
@@ -0,0 +1,45 @@
+/** @odoo-module **/
+
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+import { useComponentToModel } from '@mail/component_hooks/use_component_to_model/use_component_to_model';
+const { Component } = owl;
+
+export class AttachmentImage extends Component {
+
+    setup() {
+        super.setup();
+        useComponentToModel({ fieldName: 'component', modelName: 'mail.attachment_image', propNameAsRecordLocalId: 'attachmentImageLocalId' });
+    }
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @returns {mail.attachment_image}
+     */
+    get attachmentImage() {
+        return this.messaging && this.messaging.models['mail.attachment_image'].get(this.props.attachmentImageLocalId);
+    }
+
+}
+
+Object.assign(AttachmentImage, {
+    defaultProps: {
+        imageSize: '200x200',
```

Comment:
I'm good with just 1 prop to define image size.
Just to point out that current implementation is very prone to typos.

Maybe we could minimize this issue by having a `validate()` that checks the format?

Reply:
Alright, I added a validation based on a split on "x". Should be enough.